build.gradle

```diff
@@ -19,7 +19,13 @@ ext {
    * The version of the NYPL Audiobook API used by this package.
    */
 
-  nypl_audiobook_api_version = "0.0.66"
+  nypl_audiobook_api_version = "0.0.67"
+
+  /*
+   * The version of the NYPL AudioEngine provider used by this package.
+   */
+
+  nypl_audiobook_findaway_version = "0.0.67"
```

Comment:
Why do I need to worry about both of these in the host app now?

Reply:
We've occasionally released new versions of the API package (the API package contains the views too, remember) and that obviously meant incrementing the version number of the API package. However, that has also then meant that I had to release a new version of the Findaway implementation, even though no code changed, just so that I could increment the version number of the Findaway package to match that of the API package. I split the versions in the demo app some time ago, and I thought it was probably time to do it in the actual app too.